src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImpl.java

```diff
@@ -449,10 +449,17 @@ private void createOverrideActionPlan(
 
     // Create action plan with appropriate name and description
     String exerciseRef = collectionExercise.getExerciseRef();
+    HashMap<String, String> selectors = new HashMap<>();
```

Comment:
in Java, always prefer to use the interface rather than the concrete type

Reply:
A separate card will be created to change the type that action service receives from a `HashMap` to a `Map`